packages/curriculum-parser-markdown/plugins/insight/enki-links.js

```diff
@@ -0,0 +1,51 @@
+const visit = require('unist-util-visit');
+const { sectionNames, internalLinks } = require('@enkidevs/curriculum-helpers');
+const UrlParse = require('url-parse');
+
+const { CONTEXTS } = internalLinks;
+
+module.exports = function enkiLink() {
+  return transform;
+
+  function transform(ast) {
+    const content = ast.children.find(
+      (node) =>
+        node.type &&
+        node.type === 'section' &&
+        node.name === sectionNames.CONTENT
+    );
+    visit(content, 'link', parseEnkiLink);
+    return ast;
+  }
+
+  function parseEnkiLink(node) {
+    /**
+     * ## glossary
+     * [Object-Oriented Programming](https://enki.com/INTERNAL_LINKS_CONTEXT/path/to/entry/oop)
+     * OR
+     * [Object-Oriented Programming](enki.com/INTERNAL_LINKS_CONTEXT/path/to/entry/oop)
+     */
+    const indexOfEnki = node.url.indexOf('enki.com'); // hardcoded url for now
+    if (indexOfEnki > -1) {
```

Comment:
Should we just do:

```suggestion
    const ENKI_ROO = 'enki.com'; // hardcoded url for now
    if (node.url.startsWith(ENKI_ROOT)) {
```

Reply:
i wanted to support `https://enki.com` for some reason. Or basically, get rid of anything that comes before enki.com

Reply:
otherwise, without protocol, that `url-parse` library returns something kinda messy

Reply:
Good thinking